src/main/java/io/vertx/core/impl/Utils.java

```diff
@@ -45,4 +49,11 @@ public static boolean isWindows() {
     return isWindows;
   }
 
+  public static <E> Set<E> concurrentHashSet (){
```

Comment:
It is not necessary to add new methods.

Where a `ConcurrentHashSet` is needed, we can use `java.util.concurrent.ConcurrentHashMap#newKeySet()`

Reply:
Yes, use this method, I used it in my pekko pr too.

Reply:
Please remove this method, it is not necessary.